openhands/integrations/github/github_service.py

```diff
@@ -19,43 +19,37 @@
 
 class GitHubService:
     BASE_URL = 'https://api.github.com'
-    token: SecretStr = SecretStr('')
+    github_token: SecretStr = SecretStr('')
     refresh = False
 
     def __init__(
         self,
         user_id: str | None = None,
-        idp_token: SecretStr | None = None,
-        token: SecretStr | None = None,
+        access_token: SecretStr | None = None,
```

Comment:
wait this is still an IDP token right? I'd expect GitHubService.access_token to be a GitHub token